docs/environment-setup.md

```diff
@@ -137,67 +137,76 @@ _NOTE: This will not work on WSL (Windows Subsystem for Linux). Use the approach
+```
 
-_Note: If experiencing an error such as `Error starting userland proxy: listen tcp 0.0.0.0:6379: bind: address already in use` when using either option to start Redis. Check if an existing instance of redis is already running (Docker/Native) and stop it before starting another instance_
+Back-end will start downloading feeds and processing them until stopped. The default port # for the back-end is `3000` and can be modified with the `PORT` variable in the `.env` file. Different routes such as `http://localhost:3000/posts` and
+`http://localhost:3000/feeds` now return data. And front-end can be found in `http://localhost:8000` by default.
 
-#### Elasticsearch:
+If this doesn't work for you, it is possible that you have an old `.env` file in the root that you copied from `env.example` from telescope 1.0. Please remove it, and try again.
 
-To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
+#### Working strictly on the front-end
 
-#### Login/SSO (Optional):
+```
+cp config/env.staging .env
 
-This step is only required if you need to login to Telescope or developing/testing requiring a logged in user. Login/SSO can be started by using the command `docker-compose up login`, this will start up a container only for Login/SSO service. For more information on Login please refer to our [Login Document](login.md).
+npm run dev
+```
 
-\_Note: Users must have docker and docker-compose installed. Please refer to `Docker and Docker-Compose Set Up` earlier in this document for more information regarding Docker.
+This will provide you staging back-end without running it locally.
 
-### Finally:
+#### Want to mix and match services between local and staging
 
-Telescope requires a running back-end and front-end to start. The following steps assume you are running the back-end locally. For more information on running the front-end with our staging/production as the back-end, please refer to our [Front-End Document](gatsbyjs.md).
+See [staging-production-deployment](staging-production-deployment) for more information on running Telescope in staging or production mode.
 
-Start the back-end for Telescope, the back-end will start downloading feeds and processing them until stopped. The default port # for the back-end is `3000` and can be modified with the `PORT` variable in the `.env` file. Different routes such as `http://localhost:3000/posts` and
-`http://localhost:3000/feeds` now return data:
+This one depends on which part you're working with. For example, if you want to work with authorization, you need to specify the URL of AUTH in your `.env` file, which is, go to `.env` and find `AUTH_URL=...` and modify it to the one you want to work with. If you're testing auth locally, use `AUTH_URL=http://localhost/v1/auth`; otherwise, use the staging one, `AUTH_URL=http://dev.api.telescope.cdot.systems/v1/auth`.
```

Comment:
```suggestion
This one depends on which part you're working with. For example, if you want to work with authorization, you need to specify the URL of AUTH in your `.env` file by going to `.env` and modifying `AUTH_URL=...` to the one you want to work with. If you're testing auth locally, use `AUTH_URL=http://localhost/v1/auth`; otherwise, use the staging one, `AUTH_URL=http://dev.api.telescope.cdot.systems/v1/auth`.
```